.github/workflows/lint.yml

```diff
@@ -69,6 +69,10 @@ jobs:
                   # check for duplicate ids of formatted messages
                   pnpm run intl:extract
 
+            - name: Build Docusaurus
+              run: |
+                pnpm --filter ./docs run build
```

Comment:
Wouldn't it be a good idea to only build (= lint) the docs if there are changes to the docs?

Reply:
That would be a good idea. I couldn't find a simple way to skip the step. 

I can create a second workflow that is run only if the docs change. But with #1170 the docs have a dependency on the libraries. Meaning the libraries must be built twice in the Lint workflow and the Build Docs workflow

Reply:
We're already building the libs multiple times 😅  See release, release-canary workflows. For the build docs workflow, we should only build the necessary packages, namely `@comet/admin` packages

Reply:
Okay, I added a build docs workflow: https://github.com/vivid-planet/comet/actions/runs/5529852385/jobs/10088476382?pr=1168